ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py

```diff
@@ -283,19 +283,23 @@ def __filter_images(self, requirements: Dict[str, Any], manifest: Dict[str, Any]
         # prepare image groups:
         images = requirements['images']
         images_to_download: Dict[str, Dict] = {}
+        all_images = set()
```

Comment:
```suggestion
        selected_images: List[str] = []
```

Reply:
Good point with type hints, also the name is better. Regarding type, I would keep `set` as we are not iterating over it and we don't have to keep duplicates in it.